### PR TITLE
bpf: split complexity configurations into separate files

### PR DESCRIPTION
**Pull request by lmb**
The complexity testsuite currently loads configuration from a single file for each BPF object and kernel version. In the file each line is a distinct configuration for that BPF object. For example:

    -DFOO=1
    -DBAR=1

would compile twice. Once with FOO defined, once with BAR. In practice we have about 30 to 40 defines per each line, which makes editing, diffing, etc. a nightmare.

Switch the format from one line per configration to one file per configuration. This makes it easier to diff and also to resolve merge conflicts.

The existing configurations were converted to invidiual files using the following script:

    #!/bin/bash

    set -euo pipefail

    for dir in bpf/complexity-tests/*; do
        test -d "$dir" || continue
        pushd "$dir" || exit 2
        for f in *.txt; do
            i=0
            while read -r line; do
                test -z "$line" && continue
                i=$((i + 1))
                target="$(basename "$f" .txt)"
                mkdir -p "$target"
                echo "${line// /$'\n'}" > "$target/$i.txt"
            done < "$f"
        done
        popd || exit 2
    done

### CODE REVIEW

**Comment by lmb** [comment]
This PR is probably best reviewed with `Hide whitespace changes` enabled.

**Review by christarazi — APPROVED** [review]

File: test/verifier/verifier_test.go [hunk 1]
```diff
@@ -62,10 +63,36 @@ func getCIKernelVersion(t *testing.T) (string, string) {
...
+
+	return files
+}
+
+// readDatapathConfig turns each line in a reader into a single line with each
+// element separated by spaces.
+func readDatapathConfig(t *testing.T, r io.Reader) string {
+	scanner := bufio.NewScanner(r)
+	var lines []string
+	for i := 1; scanner.Scan(); i++ {
```
Comment (dylandreimerink) [inline test/verifier/verifier_test.go]: nit: `i` doesn't seem to be used, this can just be `for scanner.Scan() {`

**Review by dylandreimerink — APPROVED** [review]
LGTM!

**Review by rgo3 — APPROVED** [review]

**Comment by lmb** [comment]
Travis failure is due to https://github.com/cilium/cilium/issues/26617

**Comment by lmb** [comment]
Timo is on PTO, so no review from him. Dylan and Robin reviewed instead.

**Pull request merged**